### PR TITLE
Update fontstyles.html with Croatian

### PR DESCRIPTION
**Pull request by milotype**
### Type of change
- **New feature** : Croatian translation added

### CODE REVIEW

File: doc/html/fontstyles.html [hunk 1]
```diff
@@ -4,10 +4,14 @@
   <!-- AP: Last modified: 5-Oct-2006 -->
   <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8">
   <TITLE>Font Styles in various languages</TITLE>
-  <LINK REL="icon" href="fftype16.png">
-  <LINK REL="stylesheet" TYPE="text/css" HREF="FontForge.css">
+  <LINK REL="icon" href="/assets/img/old/fftype16.png">
```
Comment (jtanx) [inline doc/html/fontstyles.html]: Don't add any of this, this is auto-set when generating for fontforge.github.io

File: doc/html/fontstyles.html [hunk 2]
```diff
@@ -4,10 +4,14 @@
...
+  <LINK REL="stylesheet" TYPE="text/css" HREF="/assets/css/old/FontForge.css">
 </HEAD>
 <BODY STYLE="background: #fff">
+<div style="margin:0; height: 4	em; padding: 0.5em; background: red; color:yellow; text-align:center; font-size:1em; font-family: sans-serif;">
```
Comment (jtanx) [inline doc/html/fontstyles.html]: Same for this

Comment (milotype) [reply doc/html/fontstyles.html]: Sorry, that was taken from the fontforge.github.io repository … (I didn't check if they differ).

**Comment by milotype** [comment]
I'll send a new MR.

**Pull request closed without merging**